acquia-http-hmac.gemspec

```diff
@@ -13,7 +13,9 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.executables << 'acq-http-request'
 
+  s.add_development_dependency('rack', '~> 1')
   s.add_development_dependency('rake', '~> 10.4')
+  s.add_development_dependency('activesupport', '~> 4')
```

Comment:
@pdrakeweb The tests somewhere internally use activesupport and testing with ruby 2.1.x was failing because it tried to use activesupport 5 and that requires ruby 2.2.0+ so I've pinned it to major version 4.